website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,120 @@
+
+## Prerequisites
+* You have a Kubernetes cluster setup, this could for example be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
+* You have admin privileges to the cluster, since the Operator installation is only possible with an admin user.
+
+## Enabling Ingress
+To access an application externally, you will create _URLs_ using odo, which are implemented on a Kubernetes cluster by the ingress resources; installing an ingress-controller helps in using this feature on a Kubernetes cluster.
+
+**Minikube:** To install an ingress-controller on a minikube cluster, enable the **ingress** addon with the following command:
+```shell
+minikube addons enable ingress
+````
+To learn more about ingress addon, see [the documentation on Kubernetes website](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/).
+
+**Other Kubernetes Cluster**: To enable the Ingress feature on a Kubernetes cluster _other than minikube_, using the nginx controller see [the official nginx installation documentation](https://kubernetes.github.io/ingress-nginx/deploy/).
+
+To use a different controller, see [the ingress controller documentation](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
+
+To learn more about enabling this feature on your cluster, see the [Ingress prerequisites](https://kubernetes.io/docs/concepts/services-networking/ingress/#prerequisites) on the official kubernetes documentation.
+
+## Installing the Operator Lifecycle Manager(OLM)
+The Operator Lifecycle Manager(OLM) is a component of the Operator Framework, an open source toolkit to manage Kubernetes native applications, called Operators, in a streamlined and scalable way.[(Source)](https://olm.operatorframework.io/)
+
+[//]: # (Move this section to Architecture > Service Binding or create a new Operators doc)
+What are Operators?
+>The Operator pattern aims to capture the key aim of a human operator who is managing a service or set of services. Human operators who look after specific applications and services have deep knowledge of how the system ought to behave, how to deploy it, and how to react if there are problems.
+>
+>People who run workloads on Kubernetes often like to use automation to take care of repeatable tasks. The Operator pattern captures how you can write code to automate a task beyond what Kubernetes itself provides.
+> [(Source)](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/#motivation)
+[//]: # (Move until here)
+
+To install an Operator, we will first need to install OLM [(Operator Lifecycle Manager)](https://olm.operatorframework.io/) on the cluster.
+```shell
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/install.sh | bash -s v0.18.3
+```
+Running the script will take some time to install all the necessary resources in the Kubernetes cluster including the `OperatorGroup` resource,....
```

Comment:
Please complete the resources list

Reply:
This seems like something specific to OLM, do you think it is required?

Reply:
No, so probably just remove the 3 dots